src/SimpleCalculator/Division.cs

```diff
@@ -2,10 +2,11 @@ namespace SimpleCalculator;
 
 public class Division
 {
-    public static decimal Divide()
+    UserInputs inputs = new UserInputs();
+    public decimal Divide()
     {
-        decimal firstNumber = UserInputs.GetUserInput("Please enter the first number");
-        decimal secondNumber = UserInputs.GetUserInput("Plwease enter the second number");
+        decimal firstNumber = inputs.GetUserInput("Please enter the first number:");
```

Comment:
Could you please add a validation that `secondNumber` is not 0, otherwise `DivideByZero` exception is thrown